src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources_es.java

```diff
@@ -1,6 +1,5 @@
 /*
- * reserved comment block
- * DO NOT REMOVE OR ALTER!
+ * Copyright (c) 2019, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Remove "2019, ", same with other classes. The header (along with the LastModified tag) serves as an indication that you've modified the file. As this is the first of such modification, it just needs to be noted with the year it's done, that is "2022, ".